tests/integration/pni/search.spec.js

```diff
@@ -18,7 +18,7 @@ test.describe("PNI search", () => {
     health: 11,
     smart: 4,
     percy: 1,
-    searchTerm: 3,
+    searchTerm: 4,
```

Comment:
Hi @mmmavis! Wanted to call out that I had to update this from 3 to 4, as it appears my updates to the PNI factory file changed some of the test data being generated and is now returning 3 products instead of 4,  even when using the same RANDOM_SEED mentioned in the files comments.

Reply:
Looks good to me! Thanks for tagging me on this.